### DISCUSSION TITLE
Dual Panel

### DISCUSSION THREAD

**Discussion by RainerMathey** [aleksey-hoffman/sigma-file-manager#341] [General]
... hi, is it possible to use a dual-panel view like in MidnightCommander ???

**Comment by aleksey-hoffman** [comment]
Hi, yes, it's already implemented in v2, but v2 is not ready yet, it's in early alpha right now

**Reply by RainerMathey** [reply]
... i have v2 installed on EndeavourOS & how i can activate the dual-panel view ???

**Reply by aleksey-hoffman** [reply]
When you go to the navigator page, there's a button on top bar. But as I mentioned, v2 is not really usable yet, most features are not ported from v1 yet
